__tests__/app.test.js

```diff
@@ -85,4 +86,37 @@ describe('4 get/api/articles/:article_id', () => {
   test('when inputted a different data type, returns status 400 ', () => {
     return request(app).get("/api/articles/nonsensicalgibberish").expect(400)
     })
-  })
\ No newline at end of file
+  })
+
+
+  describe('5 get/api/articles/id/comments', () => {
+    test('returns 200, with the comments related to the article id', () => {
+        return request(app).get("/api/articles/5/comments").expect(200)
+        .then((res)=>{
+            expect(res.body.comments.length).toBe(2)
+            res.body.comments.forEach((comment)=>{
+            
+               expect(comment).toEqual(expect.objectContaining({
+                comment_id: expect.any(Number),
+                article_id : 5,
```

Comment:
The comment doesn't need to return the article id

Reply:
Isn't it a good way to check whether it's pulling the right comments? @Slyose

Reply:
<img width="528" alt="image" src="https://user-images.githubusercontent.com/99129306/202411735-5c7ca31f-f6ba-4fc1-89e8-21161451450d.png">

Reply:
@Hartooro